### PR TITLE
Adding variables for phpunit testing.

### PR DESCRIPTION
**Pull request by leonk**
### Context

https://trello.com/c/fIMUtXBN/1900-get-phpunit-tests-running-on-local-docker-environment

### Intent

Sets up environment variables required for phpunit testing.

### Considerations

Works in conjunction with PR: https://github.com/ministryofjustice/prisoner-content-hub-backend/pull/164

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

**Review by spikeheap — DISMISSED** [review]
👍

File: drupal.env.sample [hunk 1]
```diff
@@ -25,3 +25,7 @@ XDEBUG_IP=127.0.0.1
 
 # Prevent memory limit issues when installing new packages the composer
 COMPOSER_MEMORY_LIMIT=-1
+
+# Setup variables for Drupal phpunit testing.
+SIMPLETEST_BASE_URL=http://drupal:8080
```
Comment (LucasCairns) [inline drupal.env.sample:30]: The changes introduced in PR-754 change this to `prisoner-content-hub-backend`

Comment (leonk) [reply drupal.env.sample:30]: Should we rename the files as well in that case?  `drupal.env.sample` -> `prisoner-content-hub-backend.env.sample`

Comment (LucasCairns) [reply drupal.env.sample:30]: we probably should, I missed renaming those 🤦

Comment (leonk) [reply drupal.env.sample:30]: I'll update them in this PR

**Comment by leonk** [comment]
Closing this as no longer required (the environment variables can be added directly from the phpunit.xml file in the backend repo).
I cherry-picked the commit renaming the env files into a new PR https://github.com/ministryofjustice/prisoner-content-hub/pull/757

**Pull request closed without merging**